engines/wage/.gitignore

```diff
@@ -0,0 +1,3 @@
+.deps
```

Comment:
This file (.gitignore) shouldn't be here. These things are handled by the main .gitignore.